### ISSUE TITLE
Negative numbers in unread feeds.

### ISSUE THREAD

**Issue by JustPassingBy06** [yang991178/fluent-reader#216]
**Describe the bug**
When finding new articles it puts a negative number next to all articles

**To Reproduce**
I don't know what I could have done to cause this. But I have subscribed to a LOT of Youtube-feeds and a few (3) news sites.
Please say if I need to add additional info.

**Screenshots**
![image](https://user-images.githubusercontent.com/79464596/112153000-25073b00-8be3-11eb-883d-1990542c2db9.png)

**Platform (please complete the following information):**
 - OS: Windows 20H2
 - Version 1.0.0

**Comment by yang991178** [comment]
Seems like an interesting bug. Did you perform any action (mark all as read, etc.) before the negative count shows up? Does the bug persists after restarting the app?

**Comment by JustPassingBy06** [comment]
The bug persists after I restart the app. I think it could have something to do with the automatic refresher. Because if I do a manual refresh when their are no negative numbers they don't show up. If I mark all as read the numbers go away. The automatic refresher also doesn't make the numbers positive they just don't show up. When I then click an article that's marked as unread but doesn't show the number is the pop-out menu it get's negative. Sorry for bad terminology, I'm usually not talking about tech-things in English but in Dutch...

**Comment by MstMoonshine** [comment]
I met this issue as well. Interesting indeed. And the funny thing is that the negative numbers add up correctly within a group, resulting in an integer number that can be either above or below zero.

Sometimes there are unread articles even after I marked all as read, and at this time the unread number would be set to zero. Then if I click on one of the 'unread' articles, the total unread number of that group becomes -1.

I do not know how to reproduce this promisingly, but it seems to happen very often. I set the automatic refresh frequency to 10 mins and after hanging it there for a while, negative numbers would appear.

**Comment by JeppeKlitgaard** [comment]
I just wanted to add that I am seeing this as well on 1.0.1.

**Comment by polyeto** [comment]
Saw this too on 1.0.1. It happened when the feed automatically synced and I read an article at the home group. Once I checked the specific group the source of the article was located, it showed up as a -1. Interestingly, new articles that were synced did not show up at the count, but remained on boldface signifying that those were new, unread articles. I will try 1.0.2 later and see if it has been fixed.
